libethereum/TransactionQueue.cpp

```diff
@@ -100,6 +100,8 @@ ImportResult TransactionQueue::check_WITH_LOCK(h256 const& _h, IfDropped _ik)
 
 ImportResult TransactionQueue::import(Transaction const& _transaction, IfDropped _ik)
 {
+	if (t.hasZeroSignature())
+		return ImportResult::ZeroSignature;
```

Comment:
I think we cannot abort that early, at least we have to check whether the transaction is already known. Furthermore, this does not actually import the transaction. Or is that the intention?